siac/hostdbcmd.go

```diff
@@ -183,21 +184,35 @@ func hostdbcmd() {
 
 			price := host.StoragePrice.Mul(modules.BlockBytesPerMonthTerabyte)
 			downloadBWPrice := host.DownloadBandwidthPrice.Mul(modules.BytesPerTerabyte)
-			fmt.Fprintf(w, "\t%v:\t%v\t%v \t%v\t%v\t%.3f\t%s\n", len(inactiveHosts)-i, host.PublicKeyString, host.NetAddress, currencyUnits(price), currencyUnits(downloadBWPrice), uptimeRatio, scanHistStr)
+			fmt.Fprintf(w, "\t%v:\t%v\t%v\t%v\t%v\t%.3f\t%s\n", len(inactiveHosts)-i, host.PublicKeyString, host.NetAddress, currencyUnits(price), currencyUnits(downloadBWPrice), uptimeRatio, scanHistStr)
 		}
 		w.Flush()
 
+		// Grab the host at the 1/5th point and use it as the reference. (it's
+		// like using the median, except at the 1/5th point instead of the 1/2
+		// point.)
+		referenceScore := float64(1)
+		if len(activeHosts) > 0 {
+			referenceIndex := len(activeHosts) / 5
+			hostInfo := new(api.HostdbHostsGET)
+			err := getAPI("/hostdb/hosts/"+activeHosts[referenceIndex].PublicKeyString, hostInfo)
+			if err != nil {
+				die("Could not fetch provided host:", err)
+			}
+			referenceScore, _ = big.NewFloat(1).SetInt(hostInfo.ScoreBreakdown.Score.Big()).Float64()
```

Comment:
nbd, but `new(big.Float)` is our usual style

also, seems like you are converting a `big.Int` to a `big.Float` and then a `float64`, then converting other `big.Int`s to `big.Floats` and then `float64`s and then dividing them by the first `float64`. Couldn't you do the division on `big.Float`s directly? `big.Float` supports the standard printf directives, so you don't even have to change your format.